Makefile

```
@@ -281,13 +281,23 @@ api-fix-versions:
 api-doc:
 	./contrib/apiage.py --mode=write-doc
 
-api-report-updates: $(RESULTS_DIR)
+api-check-updates: $(RESULTS_DIR)
 	./contrib/apiage.py --mode=find-updates \
 		--current-tag="$$(git describe --tags --abbrev=0)" \
 		> $(RESULTS_DIR)/updates-found.json
+
+api-report-updates:
+	$(MAKE) api-check-updates
```

Why do need `$(MAKE)` here?

```suggestion
api-report-updates: api-check-updates
```
Can't we achieve the same goal with above suggestion?

---

Fixed with addition of `.PHONY:`